### PR TITLE
Bug 1193282 - No left-margin for some page titles

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

File: src/media/css/header.styl [hunk 1]
```diff
@@ -346,7 +346,11 @@ body > .banners mkt-banner {
 
 @media $base-tablet {
     .subheader {
-        h1,
+        h1 {
```
Comment (spasovski) [inline src/media/css/header.styl]: This change results in nothing. The comma operator was already ensuring the two targets share the same properties. Maybe you had a different change that was missed in the commit?

**Comment by tsl143** [comment]
The padding (left/right) is required on tablet and mobile screen, it is not required on desktop screen, so updated the required changes.

**Comment by tsl143** [comment]
@spasovski r?

File: src/media/css/header.styl [hunk 2]
```diff
@@ -448,3 +448,12 @@ body > .banners mkt-banner {
         background: url(../img/pretty/marketplace-beta-logo2X.png) no-repeat 0 50% / 200px auto;
     }
 }
+
+@media $base-desktop {
```
Comment (spasovski) [inline src/media/css/header.styl]: Your earlier fix of `padding: 40px 10px 0` was better but it needs to be within `@media $base-tablet` and not apply to the `a`. Also `0` values don't need unit types such as `px` specified.

**Comment by tsl143** [comment]
We need to apply padding on `a` on tablet and below; please check screenshot, also I removed `px`.

screen shot 2015-08-13 at 08 00 07

**Comment by tsl143** [comment]
@spasovski r?

**Comment by spasovski** [comment]
Looks good thanks! r+

**Pull request merged**